kvmagent/kvmagent/test/vm_plugin_testsuite/test_vm_dump.py

```
@@ -0,0 +1,38 @@
+    @env.test_for(handlers=[
+        vm_plugin.VmPlugin.KVM_STOP_VM_PATH
+    ])
+    @pytest_utils.ztest_decorater
+    def test_vm_stop_dump(self):
+        vm_uuid, vm = self._create_vm()
+        linux.rm_dir_force(vm_plugin.VM_CORE_DUMP_DIR)
+        linux.mkdir(vm_plugin.VM_CORE_DUMP_DIR)
+        rsp = vm_utils.stop_vm(vm_uuid, debug=True)
+
+        vmcore_dump_path = vm_plugin.VM_CORE_DUMP_DIR + "/" + vm_uuid
+        self.assertTrue(os.path.exists(vmcore_dump_path), "core dump file not exists")
```

Ruff (0.11.9)

`os` may be undefined, or defined from star imports

(F405)